app/server/controllers/UserController.js

```diff
@@ -664,29 +663,35 @@ UserController.updateRecordsWithMissingFields = function(callback) {
 }
 
 /**
- * Returns if everyone in the team has checked in
+ * Returns if the team can be assigned a table
  * @param  {Function} callback [description]
+ * @return JSON:
+ *        assign: Boolean if the team can be assigned a table or not
+ *        teammates: Id of the teammates
  */
-UserController.teamIsCompletelyCheckedIn = function(id, callback) {
+UserController.teamCanBeAssignedTable = function(id, callback) {
   this.getTeammates(id, function(err, teammates) {
     if(err) return callback(err, teammates);
 
-    let everyoneIsCheckedIn = true;
+    let numberCheckedIn = 0;
     teammates.forEach(function(member) {
-      if(!member.status.checkedIn) everyoneIsCheckedIn = false;
+      if(member.status.checkedIn) numberCheckedIn++;
     });
+    
+    Settings.getPublicSettings(function(err, settings){
+      if(err) return callback(err, settings);
 
-    return callback({}, { complete: everyoneIsCheckedIn, teammates: teammates });
+      let canBeAssigned = numberCheckedIn === settings.teamSizeAccepted;
```

Comment:
this has to be 
`numberCheckedIn >= settings.teamSizeAccepted`

for cases where there are more checked in than accepted
Eg. A team of 4 has 3 accepted while teamSizeAccepted is 4. At the same time teamSizeAccepted is changed to 3, the 4th team member checks in, so they have 4.